metadata.yaml

```diff
@@ -49,6 +49,15 @@ requires:
     limit: 1
   catalogue:
     interface: catalogue
+  certificates:
+    interface: tls-certificates
+    limit: 1
+    description: |
+      Obtain a CA and a server certificate for Prometheus to use for TLS.
+      The same cert is used for all in-cluster requests, e.g.:
```

Comment:
I would replace `The same cert is used` with `The same CA is used`; technically the CA is a cert, but the sentence is ambiguous as it hints at the Prometheus certificate being used for all in-cluster requests

Reply:
Fixed.